scripts/provisioning/s3_setup

```diff
@@ -248,35 +248,35 @@ create_s3backgroundelete_keys()
 }
 
 # Generate s3recovery keys using cipher
-create_s3recovery_keys()
-{
-    access_key=$(s3cipher --use_base64 --key_len  22  --const_key  "s3recoveryaccesskey")
-    secret_key=$(s3cipher --key_len  40  --const_key  "s3recoverysecretkey")
-}
+#create_s3recovery_keys()
+#{
+#    access_key=$(s3cipher --use_base64 --key_len  22  --const_key  "s3recoveryaccesskey")
+#    secret_key=$(s3cipher --key_len  40  --const_key  "s3recoverysecretkey")
+#}
 
 # Create s3 account for s3 recoverytool
-create_recovery_tool_account()
-{
-    output=$(ldapsearch -b "o=s3-recovery-svc,ou=accounts,dc=s3,dc=seagate,dc=com" -x -w $ldappasswd -D "cn=sgiamadmin,dc=seagate,dc=com" -H ldap://) || echo ""
-    if [[ $output == *"No such object"* ]]
-    then
-      # Create s3recovery tool account
-      create_s3recovery_keys
+#create_recovery_tool_account()
```

Comment:
Also remove call to this function, present at line 373, 374 and 375

Reply:
done